### PR TITLE
Update MTC Icons 

### PR DESCRIPTION
**Pull request by karenbtlai**
## Description

Update some MediaTransportControls icons to be the solid ones.

### CODE REVIEW

File: dev/CommonStyles/MediaTransportControls_themeresources.xaml [hunk 1]
```diff
@@ -371,29 +371,38 @@
                                                 </AppBarButton.Icon>
                                             </AppBarButton>
                                             <AppBarSeparator x:Name="LeftSeparator" Height="0" Width="0" Margin="0,0" />
-                                            <AppBarButton x:Name="StopButton" Icon="Stop" Style="{StaticResource AppBarButtonStyle}" MediaTransportControlsHelper.DropoutOrder="7" Visibility="Collapsed" />
+                                            <AppBarButton x:Name="StopButton" Icon="Stop" Style="{StaticResource AppBarButtonStyle}" MediaTransportControlsHelper.DropoutOrder="7" Visibility="Collapsed">
+                                            </AppBarButton>
```
Comment (teaP) [inline dev/CommonStyles/MediaTransportControls_themeresources.xaml:375]: I think you can revert this one -- although should it have a FontIcon just like all the others?

**Review by teaP — CHANGES_REQUESTED** [review]

File: dev/CommonStyles/MediaTransportControls_themeresources.xaml (hunk 1 above)
Comment (karenbtlai) [reply dev/CommonStyles/MediaTransportControls_themeresources.xaml]: Reverted. So turns out the solid stop button isn't part of the Segoe Fluent fonts, so using the stop symbol icon here is fine.

**Review by teaP — APPROVED** [review]

**Pull request merged**